docs/contributor/01-00-installation.md

```diff
@@ -3,9 +3,6 @@
   - [Prerequisites](#prerequisites)
```

Comment:
Can you remove the whole table of contents? I think it makes no sense to keep it when the doc contains only few sections